main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
```

Comment:
We could separate this section into a helper function where the user chooses a word for each variable. We could make the code dry by using a dictionary and a for loop that prompts the user to input a value for each key. Let's use a helper function, dictionary, and f-string to accomplish this.